run_task.py

```diff
@@ -337,17 +368,25 @@ def run(self):
 		self.workspace = folders[0]
 		self.tasks = []
 
-		project_file = OSUtils.find_file_with_pattern(self.workspace, PROJECT_FILE_NAME_PATTERN)
+		project_file = self.window.project_file_name()
+
 		if project_file:
 			with open(project_file, "r") as fp:
+
 				try:
-					tasks_json = json.load(fp)
+					json_str = fp.read()
+					# sublime supports trailing commas in the json files, json doesn't. Fix this here
+					
+					pattern = r',\s*([\]}])'
+					json_str_fixed = re.sub(pattern, r'\1', json_str)
+					tasks_json = json.loads(json_str_fixed)
 					self.tasks, parse_error_msg = TaskParser().parse_tasks(tasks_json)
 					if parse_error_msg is not None:
 						sublime.error_message(parse_error_msg)
 						return
-				except ValueError:
-					sublime.error_message(ErrorMessage.invalid_json())
+				except ValueError as error:
+					error_message = "JSON decoding error: %s " % str(error)
+					sublime.error_message(error_message)
```

Comment:
This new error message is now missing a basic information about the package that encountered an error, as well as the name of the malformed JSON file. From the user perspective, how would I know **which file** is it that needs to be fixed? To address that, the code should be changed as follows:
```python
except ValueError as error:
	sublime.error_message(ErrorMessage.invalid_json(str(error)))
```
, and the `invalid_json` method definition should be changed to:
```python
def invalid_json(json_error_message):
	error_message = PLUGIN_NAME + ': Invalid JSON. ' + ErrorMessage.CHECK_CONFIGURATION_FILE
	error_message += '\n\nError message: ' + json_error_message
	return error_message
```
This way we make sure the user gets all the information they need to fix the invalid JSON error. Also, by keeping all the error functions encapsulated in the `ErrorMessage` class, we ensure the consistent error message format across all the errors reported by Run Task.